docs/devices/TS130F.md

```diff
@@ -20,7 +20,7 @@ pageClass: device-page
 | Description | Curtain/blind switch |
 | Exposes | cover (state, position), moving, calibration, motor_reversal, backlight_mode, calibration_time, linkquality |
 | Picture | ![TuYa TS130F](https://www.zigbee2mqtt.io/images/devices/TS130F.jpg) |
-| White-label | LoraTap SC400 |
+| White-label | LoraTap SC500ZB |
```

Comment:
Could you give some more context of this change? Seems the `SC400` is also a whitelabel